app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,38 @@
  *= require_tree .
  *= require_self
  */
+@import url('https://fonts.googleapis.com/css2?family=Anton&family=Blaka&family=Lobster&family=Poppins:wght@200&family=Roboto:wght@300;700&family=Signika+Negative&display=swap');
+
+:root {
+  --light-background: #efefef;
+  --font: 'Signika Negative', sans-serif;
+}
+
+*,
+*::before,
+*::after {
+  font-family: var(--font);
+}
+
+body {
+  background-color: var(--light-background);
+}
+
+li {
+  list-style-type: none;
+}
+
+.price {
+  margin-left: auto;
+}
+
```

Comment:
- [ ] Please add more customised styles, the design should follow the gudelines: 
Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

Colors
Typography: font face, size and weight.
Layout: composition and space between elements.

Currently you have different colors on buttons and navbar 


Your design  | Behance design
------------- | -------------
<img width="330" alt="Screenshot 2022-07-01 at 13 19 51" src="https://user-images.githubusercontent.com/77038610/176865944-04abc9aa-8335-4a28-947f-ddf9a5a56064.png">  | <img width="211" alt="Screenshot 2022-07-01 at 13 20 10" src="https://user-images.githubusercontent.com/77038610/176865996-51cd1a06-6e86-40dc-8d27-8712963860e9.png">


Your design  | Behance design
------------- | -------------
<img width="330" alt="Screenshot 2022-07-01 at 13 20 31" src="https://user-images.githubusercontent.com/77038610/176866063-7246c44f-1317-4558-bf3e-6327f1ffa0d3.png"> | <img width="271" alt="Screenshot 2022-07-01 at 13 21 05" src="https://user-images.githubusercontent.com/77038610/176866190-8aec1600-f069-46a1-805a-52a4f8af960a.png">

Kindly update the styles in all views

Reply:
Done...
![CaptureIII](https://user-images.githubusercontent.com/83566118/177010707-600103ef-d2b0-4ace-8e7e-1fe894d6ac8f.PNG)